### PR TITLE
Update the SM desktop modeler instructions 

### PR DESCRIPTION
**Pull request by conceptualshark**
## Description

Closes https://github.com/camunda/developer-experience/issues/400#issue-2566470578

Adds some additional information about how the user could find the client ID, secret, and oauth URL when deploying diagrams in SM Desktop Modeler.

Also combines previous step 4 and 5, two auth methods, into a single step with a tab selector.

Will backport once changes are appropriate.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by jessesimpson36** [comment]
@conceptualshark ,  These changes look good, but I wonder if we now need to account for the difference in this form between the web modeler and desktop modeler.   The web modeler will now use user-auth to deploy models, so I'm not sure it has that form to input the client id and secret anymore.  But the desktop modeler still inputs those details. that said, the docs section is the Desktop Modeler section, so maybe that isn't going to be confusing.

**Review by jessesimpson36 — DISMISSED** [review]

**Comment by conceptualshark** [comment]
@jessesimpson36 I think since it's nested in the Desktop Modeler deploy steps it shouldn't be too much of a change, and I hope that the docs updates for configuring clusters in web modeler help make it clear for those users. With the sourcing for the input looking accurate, I've backported for @camunda/tech-writers review.

File: docs/self-managed/modeler/desktop-modeler/deploy-to-self-managed.md [hunk 1]
```diff
@@ -4,46 +4,69 @@ title: Deploy diagram
...
+   | Client secret   | The password of your Zeebe client.   | `zecret`                                                                                  |
+   | OAuth token url | The full path to the token endpoint. | `https://<keycloak base url>/auth/realms/camunda-platform/protocol/openid-connect/token`. |
+   | OAuth audience  | The permission name for Zeebe.       | `zeebe-api`                                                                               |
 
-![oauth configuration](./img/deploy-with-oauth.png)
+   </TabItem>
+   </Tabs>
 
-6. Select the **Remember** checkbox if you want to locally store the connection information.
+5. Select the **Remember** checkbox if you want to locally store the connection information.
```
Comment (mesellings) [inline docs/self-managed/modeler/desktop-modeler/deploy-to-self-managed.md:68]:
```suggestion
5. Select the **Remember** checkbox to store the connection information locally.
```
Rewording suggestion

File: docs/self-managed/modeler/desktop-modeler/deploy-to-self-managed.md [hunk 2]
```diff
@@ -4,46 +4,69 @@ title: Deploy diagram
...
+   ![basic auth configuration](./img/deploy-with-basic-auth.png)
 
-![deployment via Camunda 8](./img/deploy-endpoint.png)
+   </TabItem>
 
-4. Select **Basic**, and input your username and password in case your gateway requires basic authentication:
+   <TabItem value='oauth'>
 
-![basic auth configuration](./img/deploy-with-basic-auth.png)
+   For **OAuth**, input the credentials for your OAuth provider. These are configured as part of the default [Helm installation](/self-managed/setup/install.md) and can be discovered in [Identity](/self-managed/identity/what-is-identity.md), or are set by Zeebe [environment variables](/self-managed/zeebe-deployment/security/client-authorization.md#environment-variables).
```
Comment (mesellings) [inline docs/self-managed/modeler/desktop-modeler/deploy-to-self-managed.md:54]:
```suggestion
   For **OAuth**, input the credentials for your OAuth provider. These are configured as part of the default [Helm installation](/self-managed/setup/install.md) and can be discovered in [Identity](/self-managed/identity/what-is-identity.md), or set by Zeebe [environment variables](/self-managed/zeebe-deployment/security/client-authorization.md#environment-variables).
```
Rewording suggestion

File: docs/self-managed/modeler/desktop-modeler/deploy-to-self-managed.md [hunk 3]
```diff
@@ -4,46 +4,69 @@ title: Deploy diagram
...
+   [
+   {label: 'Basic', value: 'basic' },
+   {label: 'OAuth', value: 'oauth' }
+   ]}>
 
-:::caution
+   <TabItem value='basic'>
 
-Multi-tenancy is only available with authentication enabled [through Identity](/self-managed/identity/what-is-identity.md), and [enabled in all required components](/self-managed/concepts/multi-tenancy.md).
+   For **basic authentication**, input your username and password:
```
Comment (mesellings) [inline docs/self-managed/modeler/desktop-modeler/deploy-to-self-managed.md:46]:
```suggestion
   For **basic authentication**, enter your username and password:
```
Input or enter? Just a suggestion

File: docs/self-managed/modeler/desktop-modeler/deploy-to-self-managed.md [hunk 4]
```diff
@@ -4,46 +4,69 @@ title: Deploy diagram
...
+
+   :::caution
+
+   Multi-tenancy is only available with authentication enabled [through Identity](/self-managed/identity/what-is-identity.md), and [enabled in all required components](/self-managed/concepts/multi-tenancy.md).
+
+   :::
+
+   ![deployment via Camunda 8](./img/deploy-endpoint.png)
+
+4. Select your authentication method, and input the required credentials:
```
Comment (mesellings) [inline docs/self-managed/modeler/desktop-modeler/deploy-to-self-managed.md:36]: Input or enter? Just checking input is the correct term here 👍

**Review by mesellings — APPROVED** [review]
Lgtm, just a few non-blocking suggestions, feel free to ignore 🚀

**Comment by conceptualshark** [comment]
going to go ahead and merge this one as-is, and will make a note to review the entirety of content per suggestions in a follow-up issue.

**Pull request merged**